bind_test.go

```diff
@@ -317,14 +313,14 @@ func TestBindUnmarshalParam(t *testing.T) {
 	err := c.Bind(&result)
 	ts := Timestamp(time.Date(2016, 12, 6, 19, 9, 5, 0, time.UTC))
 
-	assert := assert.New(t)
-	if assert.NoError(err) {
+	assertion := assert.New(t)
```

Comment:
please make sure that there are no `assert.New(t)` left in any of the tests

Reply:
Ok, I will check that